### PR TITLE
Add first name to date of birth page

### PR DESCRIPTION
**Pull request by brdunfield**
https://trello.com/c/X1PE7T6X/451-s-put-the-first-name-of-the-tax-filer-on-the-sin-page-and-the-dob-page

Pretty simple change, not sure explanation is needed.

### CODE REVIEW

File: views/login/dateOfBirth.pug [hunk 1]
```diff
@@ -7,11 +7,15 @@ block variables
...
   -var dobMonth = hasData(body, 'dobMonth') ? body.dobMonth : (hasData(data.login, 'dobMonth') ? data.login.dobMonth : '')
   -var dobYear = hasData(body, 'dobYear') ? body.dobYear : (hasData(data.login, 'dobYear') ? data.login.dobYear : '')
+  -var firstName = hasData(data, 'login.firstName') ? data.login.firstName : (hasData(data, 'personal.firstName') ? data.personal.firstName : '' )
 
 block content
 
   h1 #{title}
 
+  if firstName
+    h2 #{firstName}, #{__('thanks for your social insurance number')}.
```
Comment (pcraig3) [inline views/login/dateOfBirth.pug]: Let's capitalise the 'S' like on the open/close component.

image

File: locales/en.json [hunk 2]
```diff
@@ -80,7 +80,7 @@
   "For Example:": "For Example:",
   "Continue": "Continue",
   "Enter your date of birth": "Enter your date of birth",
-  "Please enter your date of birth in the field below.": "Please enter your date of birth in the field below.",
+  "thanks for your social insurance number": "thanks for your social insurance number",
```
Comment (pcraig3) [inline locales/en.json]: Also, load the page in french so that we get the french locales key as well.

**Comment by brdunfield** [comment]
Noticed a small bug where if the language is changed during the login flow, the firstname on the pages goes away. However, app still behaves correctly and lets the user login (session data isn't lost). I'm going to see if I can fix that as part of this PR.

**Comment by brdunfield** [comment]
Nevermind, It's working fine. My brain must have been playing tricks on me.

**Pull request merged**